src/rgw/setup/rgw.py

```diff
@@ -985,19 +974,19 @@ def _update_svc_data_path_value(conf: MappedConf, client_section: str):
         svc_config_file = Rgw._get_rgw_config_path(conf)
         confstore_url = const.CONFSTORE_FILE_HANDLER + svc_config_file
         Rgw._load_rgw_config(Rgw._conf_idx, confstore_url)
-        Log.info(f'updating data_path paramter to {client_section} in {svc_config_file}')
+        Log.info(f'Updating data_path parameter to {client_section} in {svc_config_file}')
 
         # Create data path's default value e.g. /var/lib/ceph/radosgw/<cluster-id>
         data_path_default_value = const.SVC_DATA_PATH_DEFAULT_VALUE + cluster_id
         confstore_data_path_value = conf.get(const.SVC_DATA_PATH_CONFSTORE_KEY)
         if confstore_data_path_value is not None:
-           Log.info(f'Setting config key :{const.SVC_DATA_PATH_KEY} with value:{confstore_data_path_value}'
+           Log.debug(f'Setting KV pair {const.SVC_DATA_PATH_KEY}:{confstore_data_path_value}'
                f'at {client_section} section')
            Conf.set(Rgw._conf_idx, f'{client_section}>{const.SVC_DATA_PATH_KEY}', str(confstore_data_path_value))
         else:
-           Log.info(f'GConfig entry is missing for config key :{const.SVC_DATA_PATH_KEY}.'
```

Comment:
this would be useful  at INFO log level only, given default values being used.

Reply:
Done